src/overlays/actors/ovl_En_Mm/z_en_mm.c

```diff
@@ -15,13 +15,11 @@ void EnMm_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnMm_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnMm_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80965D3C(EnMm* this, GlobalContext* globalCtx);
+void func_80665D3C(EnMm* this, GlobalContext* globalCtx);
```

Comment:
Did you mean to change the 9 to a 6 (809 vs 806)? `func_80665D3C` would not be in this overlay.

Reply:
Shoot, no I did not. Didn't notice that, must have just mistyped at some point.